gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/api/spring/user/SpringDynamicUserRoleRepository.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.repository.jdbc.management.api.spring.user;
+
+import io.gravitee.am.repository.jdbc.management.api.model.JdbcUser;
+import io.reactivex.Flowable;
+import org.springframework.data.r2dbc.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.repository.reactive.RxJava2CrudRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Repository
+public interface SpringDynamicUserRoleRepository extends RxJava2CrudRepository<JdbcUser.Role, String> {
+    @Query("select * from dynamic_user_roles r where r.user_id = :user")
+    Flowable<JdbcUser.Role> findByUserId(@Param("user") String userId);
```

Comment:
:thinking: Should it be JdbcUser.DynamicRole ?

Reply:
FP